### DISCUSSION TITLE
Heat redistribution

### DISCUSSION THREAD

**Discussion by sonithls** [phoebe-project/phoebe2#621] [Questions]
1) Is it possible to set redistribution_method in phoebe? Also how to set  "frac_refl_globalredist_bol" in phoebe 2.4 
(https://github.com/phoebe-project/phoebe2/issues/95)

2) Is there any tutorial available on heat redistribution? 

I am trying to forward model the light curve of wide binaries with a cool giant as the primary star and a WD as the secondary. These systems show a wavelike variation. So, I am trying to account for the reflection effect and heat redistribution to reproduce the light curve.

**Comment by kecnry** [comment]
Unfortunately heat redistribution was never fully integrated into the python frontend and so never made it in to a released version of PHOEBE.  It is on our list to try to get back to someday, but would probably take quite a bit of effort to bring up-to-date with the rest of the code.

-Kyle

**Reply by sonithls** [reply]
Thank you for your prompt reply.
